plugins/SitesManager/SitesManager.php

```diff
@@ -61,6 +61,7 @@ public function registerEvents()
             'Request.dispatch'                       => 'redirectDashboardToWelcomePage',
             'Template.noDataPageGTMTabInstructions'  => 'noDataPageGTMTabInstructions',
             'Template.noDataPageWordpressTabInstructions'  => 'noDataPageWordpressTabInstructions',
+            'Template.noDataPageVueTabInstructions'  => 'noDataPageVueTabInstructions',
```

Comment:
We should render that using a hook only if it's required to overwrite it somewhere. Guess this isn't the case here, right?

Reply:
Yes its not needed removed this hook